slack/command.py

```diff
@@ -602,6 +573,38 @@ def call_function(command_text, context):
         return slack_response(ERR_NO_FUNC_FOUND)
 
 
+def validate_input(text, command_response):
+    """
+    Validates user input and formats it for processing
+    """
+    parts = text.split()
+    setting = format_input(parts[0].lower())
+
+    if setting not in (
+        'url',
+        'baseurl',
+        'url_pattern',
+        'url_separator',
+    ):
+        return slack_response(ERR_SET_SETTING_NOT_RECOGNISED + command_response)
+
+    return (parts, setting)
+
+
+def format_input(setting):
+    """
+    Formats valid input setting to db keyword
+    """
+    options = {
```

Comment:
could we move this to a const at the top of the file, then we can reference it both here and in the `validate_input` function